lib/asciidoctor/opal_ext/kernel.rb

```diff
@@ -1,6 +1,8 @@
 module Kernel
+  # Redefine Kernel#freeze to suppress "not supported" warning message from Opal.
+  # TODO Remove once https://github.com/opal/opal/issues/1253 is resolved.
```

Comment:
I'm 100% in support of that. Whenever you upgrade Opal and it takes over control of one of our overrides, feel free to remove it at the same time.